types/filter.d.ts

```diff
@@ -7,8 +7,13 @@ export { FilterType } from "../src/filter";
 
 export declare type SingleValue = string | number | boolean | Date;
 export declare type RangeDataType = number | Date;
+export declare type Param = {
+    value: string;
+    prependTable: boolean;
+}
+export declare type ParamListValue = Param[];
 export declare type RangeValue = Range<RangeDataType>;
-export declare type FilterValue = SingleValue | RangeValue;
+export declare type FilterValue = SingleValue | RangeValue | ParamListValue;
```

Comment:
>ParamListValue [](start = 61, length = 14)

Could this just be SingleValue[] ?

Reply:
I need to be able to prepend the table to some but not all parameters so this can't just be singleValues - but I can update params to include singleValues

---
In reply to: [260922020](https://github.com/ssivov/filter-conditions/pull/14#discussion_r260922020) [](ancestors = 260922020)